libraries/joomla/document/html/html.php

```
@@ -484,6 +484,23 @@ public function countModules($condition)
 	{
 		$operators = '(\+|\-|\*|\/|==|\!=|\<\>|\<|\>|\<=|\>=|and|or|xor)';
 		$words = preg_split('# ' . $operators . ' #', $condition, null, PREG_SPLIT_DELIM_CAPTURE);
+
+		// $words must be odd, an even number of words is a mistake so skip processing
+		if (!(count($words) & 1)) {
+			return false;
```

I'd throw an InvalidArgumentException here.